src/Compilers/CSharp/Portable/Compilation/CSharpDiagnosticFilter.cs

```diff
@@ -186,6 +187,11 @@ internal static class CSharpDiagnosticFilter
                 // 3. Compilation level
                 isSpecified = true;
             }
+            else if (syntaxTreeOptions is object && syntaxTreeOptions.TryGetGlobalDiagnosticValue(id, out report))
+            {
+                // 4. Global analyzer config options
```

Comment:
Crazy specific NIT: the rest of the comments match up with the comment verbiage at the top of the method. 

```suggestion
                // 4. Global analyzer config level
```